swift/StableDiffusion/pipeline/StableDiffusionPipeline.Configuration.swift

```diff
@@ -27,7 +27,7 @@ public struct PipelineConfiguration: Hashable {
     /// Fraction of inference steps to at which to start using the refiner unet if present in `textToImage` mode
     /// Must be between 0 and 1
     /// Higher values will result in fewer refiner steps
-    public var refinerStart: Float = 0.7
+    public var refinerStart: Float = 0.8
```

Comment:
What do you think should be the default here? I see 0.7 [here](https://github.com/huggingface/diffusers/blob/ed507680e35b7628bc11235255cfc58ad1101626/src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl_img2img.py#L656), but `0.8` in the examples [here](https://huggingface.co/docs/diffusers/using-diffusers/sdxl#refine-image-quality)

Reply:
This was based on my own testing, but according to the paper it was specialized on the first 200 steps, so 20% seems reasonable. I noticed fewer "wrong" artifacts show up with `0.8` which seems preferred as a default, whereas I didn't notice much more "refinement" with a higher percentage of refinement steps.

Reply:
Awesome, thanks for the clarification!